content/articles/how-to-integrate-lipa-na-mpesa-in-php/index.md

```diff
@@ -0,0 +1,294 @@
+### How to Integrate Safaricom Lipa Na Mpesa (STK Push) in PHP
+
+Daraja API is an Application Programming Interface that creates a bridge for payment integration to web apps and mobile apps.
+
+
+In your web app, you can create a button that when a user clicks, he/she can initiate payment. It has a button in which the user clicks it to send a request to Safaricom Daraja On receiving the request, Daraja sends an STK push which contains the amount that will be deducted from the Mpesa account. The user is prompted to input his/her PIN. Once the user inputs PIN, the transaction is processed. The amount is deducted from the Mpesa balance.
+
+
+### Prerequisites
+
+- Have some good knowledge of PHP
+
+- Have some basics of HTML and CSS
+
+- How to use XAMPP and VsCode
+
+
```

Comment:
Fix spacings, seems they have been repeated a couple of times...

Reply:
i am sorry for not replying the text on time since I have been offline. let me work on it now